bin/ansible

```diff
@@ -134,9 +134,9 @@ if __name__ == '__main__':
             # Show raw stacktraces in debug mode, It also allow pdb to
             # enter post mortem mode.
             raise
-        have_cli_options = cli is not None and cli.options is not None
+        have_cli_options = context.CLIARGS is not None
```

Comment:
Is `context.CLIARGS` ever `None`?  Looks like it's created as `CLIARGS = CLIArgs({})`

Reply:
Good catch.  It was None in an earlier version.  I'll change this conditional to also catch empty CLIArgs.

Reply:
Fixed in latest version